README.md

```diff
@@ -31,6 +31,13 @@ types of particle simulations.
 - [HOOMD-blue benchmark scripts](https://github.com/glotzerlab/hoomd-benchmarks):
   Scripts to evaluate the performance of HOOMD-blue simulations.
 
+## Related tools
+
+- [freud](https://freud.readthedocs.io/):
+  Analyze HOOMD-blue simulation results with the **freud** Python library.
+- [signac](https://signac.readthedocs.io/):
```

Comment:
I'd recommend pointing to the project homepage https://signac.io/ here. If referring to the docs, please use https://docs.signac.io/.